charter.md

```diff
@@ -1,52 +1,124 @@
+  the Steering Committee.
+
+## Direct responsibilities of the Steering Committee
+
+The following responsibilities belong to the Steering Committee.
+
+* Act as escalation point of last resort for decisions important to the 
+  project that cannot be resolved within SIGs or Committees.
+* Delegate project responsibilities and ownership to SIGs and Committees.
+* Define, evolve, and defend the non-technical vision / mission and values
+  of the project.
+* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
+* Define and enforce requirements for community groups<sup>[3](#footnote3)</sup>
+  to be in good standing such as having an approved charter.
+* Through Charters, decide which parties<sup>[2](#footnote2)</sup> are 
+  responsible for non-technical areas of the project.
+* Define and evolve guidance for group<sup>[3](#footnote3)</sup> governance
+  structures and policies, including how contributors become 
+  committers/maintainers, approvers, reviewers, members, etc.  As well as 
+  responsibilities associated with these roles.
 * Charter and refine policy for defining new community groups (including
   Special Interest Groups, Working Groups, and Committees), and establish
-  transparency and accountability policies for such groups
-* Define, evolve, and defend a Code of Conduct, which must include a neutral,
-  unbiased process for resolving conflicts
-* Define and evolve project governance structures and policies, including how
-  contributors become committers/maintainers, approvers, reviewers, members,
-  etc.
+  transparency and accountability policies for such groups.
+  
+### Not yet delegated responsibilities
+
+The following responsibilities may be delegated in the future.
+
+* Manage the Kubernetes brand to decide which things can be called “Kubernetes”
+  and how that mark can be used in relation to other efforts or vendors.
 * Decide, for the purpose of elections, who is a member of standing of the
-  Kubernetes project, and what privileges that entails
-* Decide which sub-projects are part of the Kubernetes project, including
-  accepting new sub-projects and pruning existing sub-projects to maintain
-  community focus
+  Kubernetes project, and what privileges that entails.
+* Control and delegate access to and establish processes regarding
+  any Kubernetes repository<sup>[4](#footnote4)</sup> 
+* Control and delegate access to and establish processes regarding
+  project resources/assets<sup>[5](#footnote5)</sup>  
+
+## Delegated responsibilities of note
+
+Following are noteworthy responsibilities that have been delegated and *should
+be present in the relevant charters*.  Additional delegated responsibilities 
+should also be specified in a charter document for the 
+party<sup>[2](#footnote2)</sup>.
+
+### SIG Architecture
+
+* Define, evolve, and defend the technical vision / mission and scope of the
+  project.
+* Decide which SIG is responsible for each technical area of the 
```

Comment:
I think SIG Arch has only indirect influence over this, by
* deciding what is and is not in scope for the project as a whole
* deciding where some functionality belongs in the system architecturally